charts/ocean-network-client/values.yaml

```diff
@@ -30,3 +30,9 @@ resources:
 # Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
 tolerations:
   - operator: "Exists"
+
+updateStrategy:
```

Comment:
@yehielnetapp Since we've added the updateStrategy to the values options, we need to update the documentation file as well.